AAT_android/app/src/main/java/com/aat/util/EnpointUtil.java

```diff
@@ -0,0 +1,15 @@
+package com.aat.util;
+
+import java.util.regex.Pattern;
+
+/**
+ * Created by Yesika on 1/21/2017.
+ */
+
+public class EnpointUtil {
+
+    public static String solveUrl (String url, String parameter, String value){
+        url = url.replace("{"+parameter+"}",value);
```

Comment:
I thought would be better to have something more "general".  What if we change the order of the parameters in the URL?. Then we would need to change the order of all the url+value.